cognite/client/_api/files.py

```diff
@@ -570,9 +572,17 @@ def upload_bytes(
         returned_file_metadata = res.json()
         upload_url = returned_file_metadata["uploadUrl"]
         headers = {"Content-Type": file_metadata.mime_type}
-        self._http_client_with_retry.request(
+        upload_response = self._http_client_with_retry.request(
             "PUT", upload_url, data=content, timeout=self._config.file_transfer_timeout, headers=headers
         )
+        if not upload_response.ok:
+            request_id = r.group(1) if (r := re.search(r"\nRequestId:*(.+?)\n", upload_response.text)) else None
```

Comment:
This will depend on the blob storage implementation (amazon s3, azure blob storage, google cloud storage).

Reply:
Removed on https://github.com/cognitedata/cognite-sdk-python/pull/1339/commits/54fe00e570eb83ad1702b915ff5625acf592bd99